config/routes.js

```diff
@@ -20,7 +22,10 @@ export default (router, passport, app) => {
   // Setting up the users api
   router.post('/users', users.create);
   router.post('/users/avatars', users.avatars);
+  api.get('/verify', middleware.verifyToken);
```

Comment:
The way `verifyToken` is used here means that it should be in a controller file and not a middleware file. By the way I don't think verifyToken is necessary, you can have a middleware that check that all the protected routes are protected and sends a 401 back for a bad token so the frontend can redirect the user to the login screen

Reply:
This is redundant here, the get route can be chained to the api instance we have on line 14. So it would look like 
```
api
  .post('bla bla')
  .get('yor route here')
```